spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/GravitinoIcebergCatalog.java

```diff
@@ -136,4 +166,42 @@ private boolean isSystemNamespace(String[] namespace)
     isSystemNamespace.setAccessible(true);
     return (Boolean) isSystemNamespace.invoke(baseCatalog, (Object) namespace);
   }
+
+  private com.datastrato.gravitino.rel.Table loadGravitinoTable(Identifier ident)
```

Comment:
could you place this method in `baseCatalog` and resued by `loadTable`?

Reply:
done.